examples/object-syntax/package.json

```diff
@@ -20,6 +20,7 @@
     "@wyw-in-js/ts-config": "workspace:*",
     "dedent": "^1.5.1"
   },
+  "main": "src/index.ts",
```

Comment:
Used to satisfy a resolver in Vite. As the package is private, it's not an issue.